File: core/src/main/java/com/graphhopper/routing/ev/Curvature.java
package com.graphhopper.routing.ev;

public class Curvature {
    public static final String KEY = "curvature";

    public static DecimalEncodedValue create() {
        // for now save a bit: ignore all too low values and set them to the minimum value instead
        return new DecimalEncodedValueImpl(KEY, 4, 0.25, 0.05,
                false, false, false);
    }
}

File: core/src/main/kotlin/com/graphhopper/routing/ev/Curvature.kt
package com.graphhopper.routing.ev

object Curvature {
    const val KEY = "curvature"

    @JvmStatic
    fun create(): DecimalEncodedValue =
        // for now save a bit: ignore all too low values and set them to the minimum value instead
        DecimalEncodedValueImpl(
            KEY, 4, 0.25, 0.05,
            false, false, false
        )
}
